docs/internals/layout_in_storage.rst

```diff
@@ -6,32 +6,43 @@ Layout of State Variables in Storage
 
 .. _storage-inplace-encoding:
 
-Statically-sized variables (everything except mapping and dynamically-sized
-array types) are laid out contiguously in storage starting from position ``0``.
+State variables of contracts are stored in storage in a compact way such
+that multiple values sometimes use the same storage slot.
+Except for dynamically-sized arrays and mappings (see below), data is stored
+contiguously item after item starting with the first state variable,
+which is stored in slot ``0``. For each variable,
+a size in bytes is determined according to its type.
```

Comment:
is 32 the size of a mapping?

Reply:
Fair enough... ``the required size``? Just "a size" sounds a bit weird to me. But also fine ;-).

Reply:
Yeah, I even wanted to suggest `the size` myself but I saw that @ekpyron already made a suggestion so I removed it.

It could say something like `the size of the space taken in the contiguous block` but that's way too long.